src/csvcubed/cli/inspect/metadataprinter.py

```diff
@@ -150,24 +151,25 @@ def get_datacube_results(self):
 
         Member of :class:`./MetadataPrinter`.
         """
-        assert isinstance(self.state, DataCubeState)  # Make pyright happier
-
-        csvw_state = self.state.csvw_state
+        assert isinstance(self.state, DataCubeInspector)  # Make pyright happier
 
         self.result_qube_components = self.state.get_dsd_qube_components_for_csv(
             self.primary_csv_url
         )
 
-        self.result_dataset_label_dsd_uri = (
-            select_csvw_dsd_dataset_label_and_dsd_def_uri(csvw_state.rdf_graph)
+        self.result_cube_table_identifiers = self.state.get_cube_identifiers_for_csv(
+            self.primary_csv_url
         )
-        self.result_cols_with_suppress_output_true = (
-            select_cols_where_suppress_output_is_true(csvw_state.rdf_graph)
+        self.result_column_definitions = self.state.get_column_definitions_for_csv(
```

Comment:
I'd call this `primary_csv_column_definitions` to avoid confusion over columns defined in any other CSVs referenced in the CSV-W.

I'd also do the same with `suppressed_columns` and `result_code_lists` below as well.